middleware/tracing/span.go

```diff
@@ -31,12 +32,16 @@ func setClientSpan(ctx context.Context, span trace.Span, m interface{}) {
 		switch tr.Kind() {
 		case transport.KindHTTP:
 			if ht, ok := tr.(http.Transporter); ok {
-				method := ht.Request().Method
-				route := ht.PathTemplate()
-				path := ht.Request().URL.Path
-				attrs = append(attrs, semconv.HTTPMethodKey.String(method))
-				attrs = append(attrs, semconv.HTTPRouteKey.String(route))
-				attrs = append(attrs, semconv.HTTPTargetKey.String(path))
+				attrs = append(attrs,
+					semconv.HTTPRequestMethodKey.String(ht.Request().Method),
+					semconv.HTTPRouteKey.String(ht.PathTemplate()),
+					semconv.URLFragment(ht.Request().URL.Fragment),
+					semconv.URLPath(ht.Request().URL.Path),
+					semconv.URLFull(ht.Request().URL.String()),
+					semconv.URLQuery(ht.Request().URL.RawQuery),
+					semconv.URLScheme(ht.Request().URL.Scheme),
+					semconv.UserAgentOriginal(ht.Request().UserAgent()),
+				)
```

Comment:
It seems that there is no guarantee of forward compatibility

Reply:
Maintain the original attributes to ensure backward compatibility, but also add other attributes at the same time.